index.d.ts

```diff
@@ -455,10 +455,11 @@ declare namespace Eris {
     videoQualityMode: VideoQualityMode;
   }
   interface OldMember {
-    roles: string[];
+    avatar: string | null;
     nick: string | null;
-    premiumSince: number;
     pending?: boolean;
+    premiumSince: number;
```

Comment:
Premium since can be null for old members

Reply:
That can go in its own PR, that addition isn't really an addition, it's a relocation